server/app.js

```diff
@@ -1,50 +1,65 @@
-require("dotenv").config();
-const createError = require("http-errors");
-const mongoose = require("mongoose");
-const express = require("express");
-const { join } = require("path");
-const cookieParser = require("cookie-parser");
-const logger = require("morgan");
-
-const indexRouter = require("./routes/index");
-const pingRouter = require("./routes/ping");
+require('dotenv').config();
```

Comment:
Not required actually it's present in server/bin/www

Reply:
I was not aware of this magic! Thank's for pointing that out!

Reply:
Haha no worries I also wrote it again and then removed it!